src/interactive/Popover.jsx

```diff
@@ -135,6 +116,25 @@ class Popover extends React.Component {
 		return this.menuItems;
 	}
 
+	onBodyClick(e) {
+		const isNotPopoverClick = [
+			this.menuRef,
+			this.triggerRef
+		].every(ref => !ref.contains(e.target));
```

Comment:
really trivial, but `array.includes` would be much more terse here

```js
const isPopoverClick = [this.menuRef, this.triggerRef].includes(e.target);
if (!isPopoverClick) { this.closeMenu(); }
```

definitely not important, but just wanted to make sure you know about `.some` - in theory, it will bail out of the array loop when it finds a value, so for super-long arrays, it's a better option than `.every` checking for the negated condition.